rules/windows/process_creation/proc_creation_win_susp_psloglist.yml

```
@@ -23,18 +24,24 @@ detection:
         - Image|endswith:
             - '\psloglist.exe'
             - '\psloglist64.exe'
-    selection_flgs:
+    selection_cli_eventlog:
         CommandLine|contains|all:
```

```suggestion
        CommandLine|contains:
```
The 3 in a single raw is a high spot , I can run 1 at a time